test

```diff
@@ -50,8 +50,9 @@ if [ -n "${csplit}" ] && [ -n "${head}" ]; then
     trap 'rm -r tmpdocs' EXIT
     # Create files-dir contents expected by configs
     mkdir -p tmpdocs/files-dir/tree
-    touch tmpdocs/files-dir/{config.ign,ca.pem,example.conf,example.service,file,file-epilogue,local-file3}
+    touch tmpdocs/files-dir/{config.ign,ca.pem,example.conf,example.service,file,file-epilogue,local-file3,proxy.conf}
```

Comment:
`proxy.conf` isn't needed here anymore, since we're reading from `example.conf`.